protocol/flatbuffers/all.fbs

```diff
@@ -1,5 +1,44 @@
 // This file lets us feed only a single file to flatc and is necessary to work
 // around https://github.com/google/flatbuffers/issues/6800
 
-include "server.fbs";
-include "firmware.fbs";
+include "data_feed.fbs";
+include "rpc.fbs";
+
+union InboundUnion {
+    // ---- RPC API ----
+    slimevr_protocol.rpc.HeartbeatRequest,
+    slimevr_protocol.rpc.ResetRequest,
+    slimevr_protocol.rpc.AssignTrackerRequest,
+    slimevr_protocol.rpc.SettingsRequest,
+    slimevr_protocol.rpc.ChangeSettingsRequest,
+
+    // ---- Data Feed API ----
+    slimevr_protocol.data_feed.PollDeviceStatus,
+    slimevr_protocol.data_feed.DataFeedRequest,
+    slimevr_protocol.data_feed.DataFeedNotify,
+    slimevr_protocol.data_feed.DeviceStatus,
+    slimevr_protocol.data_feed.DataFeed,
+}
+
+union OutboundUnion {
```

Comment:
We probably need some kind of Handshake packet to signal the supported operations / what kind of application we are (SteamVR driver, Firmware, Feeder, FBT Consumer).